### PR TITLE
Updating FSharp.Core packages

### LINKED ISSUE #1523: Support F# 4.1?
Thoughts on bumping FSharp.Core to latest nuget? Looks like things are set to 4.0 and with no binding redirect access, there's no way to address this ourselves unless I've missed something.

Changing binding redirect to 4.4.1.0 for the CLI caused the precompiled function app to work locally at least.

### PR DESCRIPTION
**Pull request by fabiocav**
Resolves #1523

### CODE REVIEW

File: src/WebJobs.Script/WebJobs.Script.csproj [hunk 1]
```diff
@@ -49,14 +49,12 @@
   <ItemGroup>
     <Reference Include="FSharp.Compiler.Service, Version=9.0.1.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (dsyme) [inline src/WebJobs.Script/WebJobs.Script.csproj]: I think this also need to be updated to the latest FSharp.Compiler.Service

**Comment by dsyme** [comment]
@fabiocav I added one comment - I think you also need to update to the latest FSHarp.COmpiler.Service to get all F# 4.1 features

**Comment by fabiocav** [comment]
@dsyme good to know that would be a requirement.

I did update the compiler as part of these changes, but ended up reverting those changes as there seemed to be breaking changes that were easy enough to deal with, but after noticing a behavior change (with referenced scripts using #load) I thought it would be safer to push that separately.

I'll get a PR with an update together and notify you so I can get some feedback, if you don't mind.

Thanks!

**Comment by duncanmak** [comment]
Is this still open then? I just hit this because I deployed a line of code that depended on an addition in F# 4.1 but failed in production because it's still targeting 4.0.

**Comment by fabiocav** [comment]
@duncanmak This was unfortunately reverted and is not something that will be easily addressable in 1.0 as the update conflicts with other runtime dependencies we can't safely update. We're targeting 2.0 for these updates.

**Comment by marcpiechura** [comment]
@fabiocav is 4.1 already available in the 2.0 preview ?

**Comment by ed-ilyin** [comment]
Where is the issue about adding F# 4.1 support again?

**Pull request merged**